app/serializers/spree/v2/storefront/cart_serializer_decorator.rb

```diff
@@ -0,0 +1,15 @@
+module Spree
+  module V2
+    module Storefront
+      module CartSerializerDecorator
+        def self.prepended(base)
```

Comment:
don't use this. this load extra query. Remove this from VPago.

Reply:
use order number to generate payment link instead to avoid decorating cart serializer